master/internal/db/postgres.go

```diff
@@ -61,7 +60,7 @@ func initTheOneBun(db *PgDB) {
 	// theOneBun.AddQueryHook(bundebug.NewQueryHook(bundebug.WithVerbose(true)))
 
 	// This will print only the failed queries.
-	theOneBun.AddQueryHook(bundebug.NewQueryHook())
+	// theOneBun.AddQueryHook(bundebug.NewQueryHook())
```

Comment:
I think ideally we still want debug errors for other errors

If it is too much work to do this probably is okay leaving the debug logs including the context canceled errors.